SDVModTest/UIElements/ShopHarvestPrices.cs

```diff
@@ -42,7 +42,8 @@ private void OnRenderedActiveMenu(object sender, RenderedActiveMenuEventArgs e)
             // draw shop harvest prices
             if (Game1.activeClickableMenu is ShopMenu menu)
             {
-                if (typeof(ShopMenu).GetField("hoveredItem", BindingFlags.Instance | BindingFlags.NonPublic).GetValue(menu) is Item hoverItem)
+                if (typeof(ShopMenu).GetField("hoveredItem", BindingFlags.Instance | BindingFlags.NonPublic) != null 

```

Comment:
I appreciate the fix, but I don't like that this makes it do all that work twice. An easy way to fix this would be to just have it use the ?. operator.